src/java.net.http/share/classes/jdk/internal/net/http/Stream.java

```diff
@@ -472,16 +481,19 @@ private boolean checkRequestCancelled() {
     void incoming(Http2Frame frame) throws IOException {
         if (debug.on()) debug.log("incoming: %s", frame);
         var cancelled = checkRequestCancelled() || closed;
+//        endStreamSeen = endStreamSeen || frame.getFlag(HeaderFrame.END_STREAM);
```

Comment:
leftover to be removed?

Reply:
Correct, from an older iteration. Will remove